src/strategies/completion.zsh

```diff
@@ -45,8 +45,6 @@ _zsh_autosuggest_capture_completion_widget() {
 zle -N autosuggest-capture-completion _zsh_autosuggest_capture_completion_widget
 
 _zsh_autosuggest_capture_setup() {
-	autoload -Uz is-at-least
```

Comment:
There is now one more is-at-least autoload that could possibly be cleaned up

https://github.com/zsh-users/zsh-autosuggestions/blob/aa05920a4aa5a54c2aadbafe66771d1874671e11/src/async.zsh#L47-L47